cli/config.go

```diff
@@ -108,7 +109,8 @@ type Config struct {
 }
 
 type Asset struct {
-	AdditionalTypes []string `mapstructure:"additional_types"`
+	AdditionalTypes     []string      `mapstructure:"additional_types"`
+	DeleteAssetsTimeout time.Duration `mapstructure:"delete_assets_timeout" default:"5m"`
```

Comment:
need to add validation if the time out is not specified. because if that's the case, the value will be zero.